### PR TITLE
Tt/get schedules

### PR DESCRIPTION
**Pull request by timothytong**
https://github.com/PeterChu2/WATPlanner/issues/12

### CODE REVIEW

File: app/src/main/java/com/example/peterchu/watplanner/Database/DatabaseHandler.java [hunk 1]
```diff
@@ -335,8 +381,34 @@ public Course getCourse(int id) {
...
+        courseComponent.setEnrollmentTotal(cursor.getInt(6));
+        courseComponent.setWaitingCapacity(cursor.getInt(7));
+        courseComponent.setWaitingTotal(cursor.getInt(8));
+        courseComponent.setType(cursor.getString(9));
+        courseComponent.setSection(cursor.getString(10));
+        courseComponent.setStartTime(cursor.getString(11));
+        courseComponent.setEndTime(cursor.getString(12));
+        courseComponent.setIsCancelled(cursor.getInt(13) == 1 ? true : false);
+        courseComponent.setIsClosed(cursor.getInt(14) == 1 ? true : false);
+        courseComponent.setIsTba(cursor.getInt(15) == 1 ? true : false);
```
Comment (yongla) [inline app/src/main/java/com/example/peterchu/watplanner/Database/DatabaseHandler.java]: nit: I think we can just do
```
courseComponent.setIsCancelled(cursor.getInt(13) == 1);
courseComponent.setIsClosed(cursor.getInt(14) == 1);
courseComponent.setIsTba(cursor.getInt(15) == 1);

File: app/src/main/java/com/example/peterchu/watplanner/home/HomePresenter.java [hunk 2]
```diff
@@ -95,24 +99,49 @@ public void onFailure(Call<CourseResponse> call, Throwable t) {
...
+                    public void onFailure(Call<CourseScheduleResponse> call, Throwable t) {
+                        // Log error here since request failed
+                        Log.e("HomePresenter", t.toString());
+                    }
+                });
+            } else {
+                Log.d("MyActivity", "Num schedules in db: " + dbHandler.getSchedulesCount());
+                List<CourseComponent> schedules = dbHandler.getCourseSchedule("ECE", "103");
+                for (CourseComponent c : schedules) {
+                    Log.d("MyActivity", c.toString());
```
Comment (yongla) [inline app/src/main/java/com/example/peterchu/watplanner/home/HomePresenter.java]: Is this else statement necessary? Looks like it's just for debugging purposes and not needed for master

**Review by yongla — CHANGES_REQUESTED** [review]
Just some minor stuff

**Comment by timothytong** [comment]
Comments addressed

**Review by yongla — APPROVED** [review]
lets get it!

**Pull request merged**